### PR TITLE
feat(web): album and face menu dots visible on hover only

### PR DESCRIPTION
**Pull request by waclaw66**
Currently all albums and faces have context menu dots visible, that is very disruptive. Proposed change show them only when mouse hovers over album or face.
menu-dots
album-dark
faces-dark

### CODE REVIEW

File: web/src/lib/components/faces-page/people-card.svelte [hunk 1]
```diff
@@ -10,16 +10,27 @@
 
   export let person: PersonResponseDto;
 
+  let showVerticalDots = false;
   let showContextMenu = false;
-  let dispatch = createEventDispatcher<{
```
Comment (alextran1502) [inline web/src/lib/components/faces-page/people-card.svelte:14]: Please keep the types here

Comment (waclaw66) [reply web/src/lib/components/faces-page/people-card.svelte:14]: fixed

**Comment by daniele-athome** [comment]
@waclaw66 why did you close this?

**Comment by waclaw66** [comment]
@daniele-athome: I messed up commits of two pull requests together. I'll create a new PR properly.

**Comment by waclaw66** [comment]
proper PR https://github.com/immich-app/immich/pull/3859

**Pull request closed without merging**